tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/PoPTests.cs

```diff
@@ -492,7 +492,8 @@ public async Task WamUsernamePasswordRequestWithPOPAsync()
 
         private static X509Certificate2 GetCertificate()
         {
-            X509Certificate2 cert = CertificateHelper.FindCertificateByThumbprint(TestConstants.AutomationTestThumbprint);
+            X509Certificate2 cert = CertificateHelper.FindCertificateByName(TestConstants.AutomationTestCertName);
```

Comment:
nice!!! but, let's not remove `FindCertificateByThumbprint` method from our code completely.

Reply:
Why not remove it, We are only looking for the subject name now.

Reply:
Simple way to tell if you should remove a method - are there 0 references to it? If yes, then remove it :)